mozdef_util/mozdef_util/elasticsearch_client.py

```diff
@@ -160,6 +166,9 @@ def save_event(self, body, index='events', doc_id=None, bulk=False):
         event.add_required_fields()
         return self.__save_document(index=index, body=event, doc_id=doc_id, bulk=bulk)
 
+    def save_dashboard(self, body, dash_name, doc_id=None):
+        return self.es_connection.index(index='.kibana', doc_type='dashboard', body=body, id=doc_id)
```

Comment:
this is incorrect for es6, it should be 'doc' the "type" is set to 'dashboard'
Otherwise, when using kibana and attempting to save a new dashboard through the interface, this is going to cause probs.

Reply:
I ended up removing this function all together as one rest api unt test was the only thing using it.